react-ui/src/predictions.js

```diff
@@ -5,7 +5,7 @@ import { StaggeredMotion, spring } from 'react-motion';
 class Predictions extends Component {
   static propTypes = {
     // The Predictive Vision Predictions
-    contents: React.PropTypes.array.isRequired
+    contents: null
```

Comment:
This seems strange to set a propType to `null`, since the point of propTypes is to enforce types.

Either [set this propType to sensible value](https://github.com/facebook/prop-types/blob/master/README.md#user-content-usage) or remove propTypes altogether.

Reply:
Ah yes, I set it to null to get around an error and then forgot to revisit the issue. The root of the problem is that React moved PropTypes into its own npm package. 

A new [commit ](https://github.com/heroku/einstein-vision-node/pull/9/commits/99c7b033b1b7716a4a0dd5337f7d5bf228a7934c) addresses this!

Reply:
Lovely! Merging it 🏁